topics/RDMA.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "RDMA support"
+linkTitle: "RDMA support"
+description: Valkey Over RDMA support
+---
+
+Valkey supports the Remote Direct Memory Access (RDMA) connection type via a
+Valkey module that can be dynamically loaded on demand.
+
```

Comment:
I suggest adding a new "Limitations" section.  It could include things like
- only on Linux
- not compatible with replication
- active connections limited by RDMA network card, unlikely to reach 10,000

Reply:
Sound good! Fix later. Thanks for all your suggestion!